### DISCUSSION TITLE
The working directory 'C:/DatabaseSeed' is invalid, it needs to be an absolute path.

### DISCUSSION THREAD

**Discussion by modmoto** [pkosiec/mongo-seeding#173] [Q&A] [answered]
## Description

I am trying to use the docker container on a windows system, but it does not seem to accept a windows file system. I tried multiple varations of "\" and the same command works on linux fine (in my CD).  

a few examples are:

```
docker run --rm --network=host -v C:\\DatabaseSeed:C:\\DatabaseSeed -w C:\\DatabaseSeed pkosiec/mongo-seeding
docker run --rm --network=host -v C:/DatabaseSeed:C:/DatabaseSeed -w C:/DatabaseSeed pkosiec/mongo-seeding
docker run --rm --network=host -v "C:/DatabaseSeed":"C:/DatabaseSeed" -w "C:/DatabaseSeed" pkosiec/mongo-seeding
```

Or am I doing something wrong here?

## Expected behavior

Database gets filled like on linux

## Actual behavior

```
Error response from daemon: the working directory 'C:\DatabaseSeed' is invalid, it needs to be an absolute path.
```

## Steps to reproduce
run on windows one of the following, with the SeedFolder being `DatabaseSeed`
```
docker run --rm --network=host -v C:\\DatabaseSeed:C:\\DatabaseSeed -w C:\\DatabaseSeed pkosiec/mongo-seeding
docker run --rm --network=host -v C:/DatabaseSeed:C:/DatabaseSeed -w C:/DatabaseSeed pkosiec/mongo-seeding
docker run --rm --network=host -v "C:/DatabaseSeed":"C:/DatabaseSeed" -w "C:/DatabaseSeed" pkosiec/mongo-seeding
```

**Comment by pkosiec** [comment, marked as answer]
Hi @modmoto,
The error `Error response from daemon: ...` comes from Docker, not from Mongo Seeding container. I think you specify the destination path incorrectly. Destination path is the path in the Linux container you run (Mongo Seeding is a container based on Linux Alpine, so it has linux filesystem on it).

Please try to use something like:

```bash
docker run --rm --network=host -v C:/DatabaseSeed:/data -w /data pkosiec/mongo-seeding
```

Unfortunately, I don't have a Windows machine to try it by myself. Please let me know if that works.

Cheers!

**Reply by modmoto** [reply]
Hey, 

thanks a lot for the quick response. That actually seemed to do the trick. 

Also thanks a lot for the tool, makes our CI/CD much slicker =D

**Reply by pkosiec** [reply]
I'm glad it works! Please mark my post as an answer. Thanks!

Enjoy using Mongo Seeding!

**Reply by modmoto** [reply]
jeah sure, never seen those GH disscussions, was not aware that this is now a thing.
